packages/expo-camera/ios/EXCamera/Utilities/EXCameraUtils.m

```diff
@@ -121,19 +135,41 @@ + (NSString *)writeImage:(NSData *)image toPath:(NSString *)path
   return [fileURL absoluteString];
 }
 
-+ (void)updateImageSampleMetadata:(CMSampleBufferRef)imageSampleBuffer withAdditionalData:(NSDictionary *)additionalData inResponse:(NSMutableDictionary *)response
++ (NSData *)writeImageToData:(UIImage *)image withExifData:(NSDictionary *)exif imageQuality:(float)quality
```

Comment:
Change this name to express accurately what this method is actually doing, please see my suggestion in other comment.